generator/osm_translator.cpp

```diff
@@ -0,0 +1,563 @@
+#include "generator/osm_translator.hpp"
+
+namespace
+{
+using namespace generator;
```

Comment:
Alternatively, this could be an anonymous namespace inside namespace generator.

Reply:
++